### PR TITLE
Lesson 3: Introduction to Express

### PR DESCRIPTION
**Pull request by IrynaKolh**
answers for lesson 3 - Introduction to Express

### CODE REVIEW

File: 02-express-tutorial/app.js [hunk 1]
```diff
@@ -1 +1,60 @@
...
+    foundProduct = foundProduct.filter((product) => {
+      return product.name.startsWith(search);
+    });
+  }
+  if (limit) {
+    foundProduct = foundProduct.slice(0, parseInt(limit));
+  }
+  // http://localhost:3000/api/v1/query?search=&limit=5&price=20   => 2 products
+  if (price) {
+    foundProduct = foundProduct.filter((product) => product.price >= 20.0);
```
Comment (milesgarrick) [inline 02-express-tutorial/app.js]: It looks like you just forgot to replace 20.0 with the price parameter.

**Review by milesgarrick — APPROVED** [review]
Excellent, excellent, excellent work!  The option button you added works great, as well.  Fix that one spot and you're golden!

**Comment by IrynaKolh** [comment]
> Excellent, excellent, excellent work! The option button you added works great, as well. Fix that one spot and you're golden!

Thank you... yes, I just foggot...  thank you that mentioned

File: 02-express-tutorial/app.js (hunk 1 above)
Comment (IrynaKolh) [reply 02-express-tutorial/app.js]: Done

**Pull request merged**